File: java-codellama7B-FP-all/taskid-27_flip_case-gen0.java
import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    // For a given string, flip lowercase characters to uppercase and uppercase to lowercase.
    // >>> flipCase(("Hello"))
    // ("hELLO")
    public static String flipCase(String string) {

        char[] flip = string.toCharArray();
        for(int i = 0; i < flip.length; i++) {
            if (flip[i] >= 'a' && flip[i] <= 'z') {
                flip[i] = (char) (flip[i] - 32);
            } else if (flip[i] >= 'A' && flip[i] <= 'Z') {
                flip[i] = (char) (flip[i] + 32);
            }
        }
        return String.valueOf(flip);
    }
    public static void main(String[] args) {
    assert(flipCase(("")).equals(("")));
    assert(flipCase(("Hello!")).equals(("hELLO!")));
    assert(flipCase(("These violent delights have violent ends")).equals(("tHESE VIOLENT DELIGHTS HAVE VIOLENT ENDS")));
    }

}

File: archive/py-codellama34B-AWQ-all/taskid-27_flip_case-gen0.py
def flip_case(string: str) -> str:
    """ For a given string, flip lowercase characters to uppercase and uppercase to lowercase.
    >>> flip_case('Hello')
    'hELLO'
    """

    flipped_chars = []
    for char in string:
        if char.islower():
            flipped_chars.append(char.upper())
        else:
            flipped_chars.append(char.lower())
    return ''.join(flipped_chars)
